### PR TITLE
Fix autodetect for 3850s

### PR DESCRIPTION
**Pull request by rlaneyjr**
Autodetect for 3850s would use cisco_ios when this should be cisco_xe.  Fixed for both SSH and SNMP.  Also added Panos running config mapper command "show config running".

### CODE REVIEW

File: netmiko/paloalto/paloalto_panos.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def commit(
             if device_and_network:
                 command_string += " device-and-network"
             if policy_and_objects:
-                command_string += " device-and-network"
+                command_string += " policy-and-objects"
```
Comment (ktbyers) [inline netmiko/paloalto/paloalto_panos.py:99]: This looks like a rebase issue.

File: netmiko/snmp_autodetect.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
     },
     "cisco_xe": {
         "oid": ".1.3.6.1.2.1.1.1.0",
-        "expr": re.compile(r".*IOS-XE Software,.*", re.IGNORECASE),
+        "expr": re.compile(r".*Catalyst L3 Switch Software.*", re.IGNORECASE),
```
Comment (ktbyers) [inline netmiko/snmp_autodetect.py:62]: hmmmm, I don't think that will work (i.e. that is too generic and would match platforms that should be categorized as `cisco_ios`)

**Comment by ktbyers** [comment]
It looks like you need to `rebase` as there are changes in your PR that shouldn't be here.

Also, it looks like your SNMP change wouldn't work. It would need to have some identifier that indicated `IOS-XE` (if possible).

File: netmiko/ssh_autodetect.py [hunk 3]
```diff
@@ -93,6 +93,12 @@
         "priority": 99,
         "dispatch": "_autodetect_std",
     },
+    "cisco_xe": {
+        "cmd": "show version",
+        "search_patterns": [r"Cisco IOS XE Software", r"IOS-EX"],
```
Comment (IceN9ne) [inline netmiko/ssh_autodetect.py:98]: Looks like a typo of `IOS-XE`?

**Comment by ktbyers** [comment]
No response on issues with the PR...so am going to go ahead and close.

**Pull request closed without merging**